Source/engine/render/cl2_render.hpp

```diff
@@ -65,6 +73,14 @@ void Cl2DrawOutlineSkipColorZero(const Surface &out, uint8_t col, Point position
  */
 void Cl2DrawTRN(const Surface &out, Point position, CelSprite cel, int frame, uint8_t *trn);
 
+/**
+ * @brief Same as Cl2DrawTRN but position.y is the top of the sprite instead of the bottom.
+ */
+inline void RenderCl2SpriteWithTRN(const Surface &out, CelFrameWithHeight cel, Point position, uint8_t *trn)

```

Comment:
Should we adapt the code to position them from the bottom? Or change rendering of other sprites?

Reply:
This PR only switches from PCX to CL2.
The code that rendered PCX required height.

In the future, we can either:

1. Require that all sprites have height (that means hardcoding the height as well).
2. Migrate the few places that need height to position from the bottom. Note that, for fonts, we'll need to know the height for regardless but that's easy to hardcode as there is only a few font sizes.

Changing that and the decision on that is perhaps out of scope for this PR.

Reply:
I REALLY do not like the idea of hard-coding the dimensions. If we go down this route we need to figure out a way to determiner the size instead.